docs/sphinx/developers/index.txt

```diff
@@ -65,6 +65,7 @@ Contributing to Bio-Formats
     :maxdepth: 1
 
     commit-testing
+    format-documentation
```

Comment:
I'd be inclined to place this below 'reader guide' - as it is, it's in the middle of a bunch of testing related pages.